pkg/kube/object.go

```diff
@@ -154,6 +162,45 @@ func GetPodSpec(obj client.Object) (corev1.PodSpec, error) {
 	}
 }
 
+func (o *ObjectResolver) GetCustomAnnotationsFromConfig(ctx context.Context, executionMode ExecutionMode) (map[string]string, error) {
```

Comment:
Sure Daniel. Will do make these suggested changes.

Reply:
I noticed one thing that only the Scanner struct for vulnerabilityreport has the ConfigData attribute wherease rest of the Scanner structs defined under the ciskubebenchreport, configauditreport and kubehunter packages don't have ConfigData as an attribute.
So, I'll refactor all of those Scanner structs to have the ConfigData attribute so that all of them at the time of scanning can leverage the `GetScanJobAnnotations()` method so as to populate the annotations of the pods spun up by the respective scan jobs.

Reply:
You're right. We only injected ConfigData to vulnerability scanner